recipes/libertem-qd-mpx/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "libertem-qd-mpx" %}
+{% set version = "0.2.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/libertem_qd_mpx-{{ version }}.tar.gz
+  sha256: 0e351192a4b78b9adbccafb1791711372fab27dd055a6bfecb74b5ff71bbb9ab
+
+build:
+  skip: true  # [py<37]
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}
+    - rust >=1.66
+    - cargo-bundle-licenses
+  host:
+    - python
+    - maturin >=1.5,<2.0
+    - numpy >=1.6.1,<2
```

NumPy should be unpinned here to take the pin from the global pinning
```suggestion
    - numpy
```